software/controller/controller.sh

```diff
@@ -129,6 +130,16 @@ configure_platformio() {
   echo "Updated udev rules. You might have to restart your machine for changes to become effective."
 }
 
+update_platformio() {
+  pio upgrade
+  pio update
+  pio system prune
+  pio platform install native
+  pio platform install ststm32
```

Comment:
These last two should auto-install during the build process, so maybe this is not necessary?
We probably should minimize the number of places where we have to add version qualifiers, in the event of an update breaking our workflow.